src/components/NumberInput/config.js

```diff
@@ -0,0 +1,13 @@
+let NumberInputConfig = {
+  DEFAULT_FONT_SIZE: 40,
+  POPUP_X: 0,
+  POPUP_Y: 0,
+  POPUP_WIDTH: 80,
+  POPUP_HEIGHT: 80,
+  POPUP_COLOR: 0xFF0eb4ea,
+  NUM_TXT_X: 10,
```

Comment:
Number should be centred both vertically and horizontally in popup square

Reply:
NumberInput text display alignment done.